config/pitest-suppressions/pitest-api-suppressions.xml

```diff
@@ -27,15 +27,6 @@
     <lineContent>severityLevel = SeverityLevel.getInstance(severity);</lineContent>
   </mutation>
 
-  <mutation unstable="false">
-    <sourceFile>FileText.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.api.FileText</mutatedClass>
-    <mutatedMethod>&lt;init&gt;</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
```

Comment:
Let's disable mutator at pom.xml  with comment until issue in pitest are resolved